js/piwik.js

```diff
@@ -2605,10 +2609,34 @@ if (typeof Piwik !== 'object') {
                 callback();
             }
 
+            /**
+             * Send each bulk tracks to the server
+             * @return {void}
+             */
+            function sendBulkTrackingRequests() {
```

Comment:
I'm wondering if the method `sendBulkRequest()` can be used here? Or maybe the name could be bit different like `sendBulkTrackingCollection()` and inside this method call `sendBulkRequest(bulkTrackingCollection)`?